internal/patches/patches.go

```diff
@@ -0,0 +1,53 @@
+package patches
+
+import (
+	"encoding/json"
+	"sort"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+	jsonpatch "gomodules.xyz/jsonpatch/v2"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/giantswarm/azure-admission-controller/pkg/mutator"
+)
+
+func GeneratePatchesFrom(originalJSON []byte, current runtime.Object) ([]mutator.PatchOperation, error) {
```

Comment:
Minor nitpick: When package is `patches`, it might make sense to drop it from the function name:
`patches.GeneratePatchesFrom()` -> `patches.GenerateFrom()`

Same below for `SkipPatchesForPath`

See: https://blog.golang.org/package-names#TOC_3.